pipeline-examples/parallel-multiple-nodes/ParallelMultipleNodes.groovy

```diff
@@ -0,0 +1,12 @@
+def labels = ['precise', 'trusty'] // labels for Jenkins node types we will build on
+def builders = [:]
+for (x in labels) {
```

Comment:
I did that and it didn't work.
12:37 <+jglick> autojack: the subject of endless duplicates on stackoverflow.com/questions/tagged/jenkins-workflow. Actually a Groovy question, not Pipeline specific. Short answer: bind `label` to a fresh variable outside the closure.